pontoon/sync/formats/ftl.py

```diff
@@ -5,9 +5,12 @@
 import logging
 import os
 
-from fluent.syntax import ast
-from fluent.syntax import parser
-from fluent.syntax import serializer
+from fluent.syntax import (
+    ast,
+    FluentParser,
+    FluentSerializer
+)
+from fluent.syntax.serializer import serialize_message
```

Comment:
It's better to use `FluentSerializer.serialize_entry` rather than `serialize_message` which is intended as a private function.